lms/static/js/bootstrap-transition.js

```diff
@@ -22,7 +22,7 @@
 
   $(function () {
 
-    "use strict"; // jshint ;_;
+    "use strict";
```

Comment:
They should. Curious that difflint still passed.

If it's not absolutely required to get this PR green, I'd prefer to keep changes like this for the coming eslint autofix pr (which will change double quotes to single)